frontend/src/components/common/ReleaseNotes/UpdatePopup.tsx

```diff
@@ -1,11 +1,22 @@
 import { Button, Snackbar } from '@material-ui/core';
 import React from 'react';
+import { useEffect, useState } from 'react';
 import { useTranslation } from 'react-i18next';
 
-function UpdatePopup(props: { releaseDownloadURL: string }) {
-  const [show, setShow] = React.useState(true);
+export interface UpdatePopupProps {
+  releaseDownloadURL: string;
+  open: boolean;
+}
+
+function UpdatePopup(props: UpdatePopupProps) {
   const { releaseDownloadURL } = props;
   const { t } = useTranslation('frequent');
+  const { open } = props;
+
+  const [show, setShow] = useState(open);
```

Comment:
Once you change the open to join it with the releaseDownloadURL var, please place this line right under the useTranslation state. (have the newline after it, not before, for readability)